source/docs/advanced/features/disable-hyration.md

```diff
@@ -0,0 +1,33 @@
+---
+id: disable-hyration
+title: Disable Hydration
+---
+
+# What is Hydration?
+
+<blockquote class="feature--new">
+_Since version 2.16_
+</blockquote>
+
+> Hydration or rehydration is a technique in which client-side JavaScript converts a static HTML web page, delivered either through static hosting or server-side rendering, into a dynamic web page by attaching event handlers to the HTML elements. Because the HTML is pre-rendered on a server, this allows for a fast "first contentful paint" (when useful data is first displayed to the user), but there is a period of time afterward where the page appears to be fully loaded and interactive, but is not until the client-side JavaScript is executed and event handlers have been attached.
+
+## Disabling Hydration
+
+While it is not recommended to disable hydration in some senarios one might want to do that. For example to disable the render cycle of the Google crawler bot.
+
+To disable hydration in Front-Commerce you need to set a variable called `__HYDRATE__` to true on the global `window` object. To do so you can add the following snippet to your template HTML file (`src/template/index.html`):
+
+```
+<script>
+  window.__HYDRATE__ = (function() {
+    var shouldHydrate = some_logic_to_determine_if_page_should_hydrate;
+    return shouldHydrate;
+  })();
+</script>
+```
+
+A typical logic to use to determine if one should hydrate or not is to inspect the `navigator.userAgent` variable to determine for example if it is the Google crawler bot.
```

Comment:
**issue:** I'm not sure this recommendation is relevant as we don't have any clue of its impact (it can lead to cloaking).

**suggestion:** we could illustrate with the example we got, to prevent hydration on different domain.\
```
window.__HYDRATE__ = location.host !== "webcache.googleusercontent.com";
```